apps/src/lib/node/ledger/shell/finalize_block.rs

```diff
@@ -512,8 +517,12 @@ where
 #[cfg(test)]
 mod test_finalize_block {
     use namada::types::address::xan;
+    use namada::types::ethereum_events::EthAddress;
     use namada::types::storage::Epoch;
     use namada::types::transaction::{EncryptionKey, Fee};
+    use namada::types::vote_extensions::ethereum_events::{
+        MultiSignedEthEvent, Vext, VextDigest,
+    };
```

Comment:
can you use `Vext` with its associated mod, for clarity? e.g. `ethereum_events::Vext`. same thing for `VextDigest`.